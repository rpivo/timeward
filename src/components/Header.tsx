import React from 'react';
import { Link } from "react-router-dom";
import StyledHeader from '@styles/components/Header.styled';

const Header = (): JSX.Element =>
  <StyledHeader>
    <h6><Link to="/dashboard">TIMEWARD</Link></h6>
    <nav>
      <Link to="/dashboard">Dashboard</Link>
      <Link to="/login">Login</Link>
    </nav>
  </StyledHeader>;

export default Header;
