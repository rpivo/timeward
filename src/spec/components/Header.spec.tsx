import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';
import Header from '@components/Header';

describe('Header', () => {

  const header =
    <MemoryRouter>
      <Header isAuthorized={true} />
    </MemoryRouter>;

  describe('render', () => {
    const testRenderer = renderer.create(header);

    it('should render correctly', () => {
      testRenderer.toJSON();
      expect(testRenderer).toMatchSnapshot();
    });

  });
});
