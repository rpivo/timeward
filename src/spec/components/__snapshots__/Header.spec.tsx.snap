// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render should render correctly 1`] = `
<header
  className="sc-AxjAm gTaTel"
>
  <h6>
    <a
      href="/dashboard"
      onClick={[Function]}
    >
      TIMEWARD
    </a>
  </h6>
  <nav>
    <a
      href="/dashboard"
      onClick={[Function]}
    >
      Dashboard
    </a>
    <a
      href="/login"
      onClick={[Function]}
    >
      Account
    </a>
  </nav>
</header>
`;
